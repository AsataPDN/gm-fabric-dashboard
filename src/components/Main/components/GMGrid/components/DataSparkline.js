import styled from "styled-components";

import { spacingScale } from "style/styleFunctions";

const DataSparkline = styled.div`
  opacity: 0.7;
  text-align: left;
  display: inline-block;
  width: 6em;
  padding-left: ${spacingScale(1)};
  height: ${spacingScale(3)};
`;

export default DataSparkline;
