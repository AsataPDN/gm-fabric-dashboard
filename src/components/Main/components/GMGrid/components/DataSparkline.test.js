import React from "react";
import { shallow } from "enzyme";
import DataSparkline from "./DataSparkline";

describe("DataSparkline", () => {
  it("matches snapshot", () => {
    const aDataSparkline = shallow(<DataSparkline />);
    expect(aDataSparkline).toMatchSnapshot();
  });
});
