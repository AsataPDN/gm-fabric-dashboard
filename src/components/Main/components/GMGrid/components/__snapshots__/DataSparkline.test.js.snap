// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataSparkline matches snapshot 1`] = `
.c0 {
  opacity: 0.7;
  text-align: left;
  display: inline-block;
  width: 6em;
  padding-left: 8px;
  height: 24px;
}

<div
  className="c0"
/>
`;
